modules/console/pages/flow-console.adoc

```diff
@@ -0,0 +1,114 @@
+include::{mod-loc}attributes.adoc[]
+ifndef::mod-loc[]
+:mod-loc: ./partials/
+endif::mod-loc[]
+//Category: flow-console
+// Type: assembly
+[id="skupper-console"] 
+= Using the Skupper console
+
+include::{mod-loc}preface.adoc[]
+
+The Skupper console provides data and visualizations of the traffic flow between Skupper sites.
+
+// Type: procedure
+[id="enabling-console"] 
+== Enabling the Skupper console
+
+By default, when you create a {skupper-name} site, you do not the Skupper console.
+
+When enabled, the Skupper console URL is displayed whenever you check site status using `skupper status`.
+
+.Prerequisites
+
+* A Kubernetes namespace where you plan to create a site
+
+.Procedure
+
+. Determine which site in your {service-network} is best to enable the console.
++
+--
+Enabling the console also requires that you enable the flow-collector component, which requires resources to process traffic data from all sites.
+You might locate the console using the following criteria:
```

Comment:
If these best practices are not fundamentals,  I think it would be better put these info on the bottom of the doc, maybe creating a section called tips

Reply:
Definitely a priority, because once you create a site, you cannot reconfigure, you gotta delete and start again. If it was at bottom, you'd be kicking yourself after putting the flow collector in wrong place